src/init.cpp

```diff
@@ -2049,8 +2047,6 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
         activeMasternodeInfo.blsPubKeyOperator = std::make_unique<CBLSPublicKey>();
     }
 
-    // ********************************************************* Step 10b: schedule Dash-specific tasks
-
```

Comment:
why this comment has to be removed? it's a messy init process, good to break it in stages